### PR TITLE
Summaries of the Dharma  Analayo

### PR DESCRIPTION
**Pull request by JJM317**
I entered this as an article. Should it be in canon with Analayo as translator instead?

### CODE REVIEW

File: _content/articles/summaries-of- the- dharma-dn-12_analayo.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Summaries of the Dharma: A Translation of Dīrgha-āgama Discourse No. 12"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/summaries.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1faTaaSYl4nZfzCjpUmp6Vo30h_PrqhwI/view?usp=drive_link"
+course: karma
+tags:
+  - agama
```
Comment (buddhist-uni) [inline _content/articles/summaries-of- the- dharma-dn-12_analayo.md:10]: `da` is the tag for the Dirgha Agama specifically

File: _content/articles/summaries-of- the- dharma-dn-12_analayo.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Summaries of the Dharma: A Translation of Dīrgha-āgama Discourse No. 12"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/summaries.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1faTaaSYl4nZfzCjpUmp6Vo30h_PrqhwI/view?usp=drive_link"
+course: karma
```
Comment (buddhist-uni) [inline _content/articles/summaries-of- the- dharma-dn-12_analayo.md:8]: let's add `status: featured` though as this is a beautiful and important sutta

File: _content/articles/summaries-of- the- dharma-dn-12_analayo.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - ethics
+year: 2013
+journal: "Asian Literature and Translation"
+publisher: "Cardiff University Press"
+volume: 1
+number: 6
+pages: "1--14"
+---
+
+A translation of the twelfth discourse in the Chinese Dīrgha-āgama with a short introduction by Bhikkhu Anālayo. This discourse is without known parallels, though it does share similarities with the Saṅgīti-sūtra and the Daśottara-sūtra.
```
Comment (buddhist-uni) [inline _content/articles/summaries-of- the- dharma-dn-12_analayo.md:20]: What did you learn from the sutta?  For me, it's a striking example of how the path karma truly is "neither light nor dark"

**Review by buddhist-uni — COMMENTED** [review]
Yeah, it really could go either way! But since there's some commentary added (and you've already done it as an article!) let's just keep it as an article 😊

File: _content/articles/summaries-of- the- dharma-dn-12_analayo.md (hunk 3 above)
Comment (JJM317) [reply _content/articles/summaries-of- the- dharma-dn-12_analayo.md:20]: I put something in that stuck out to me. I can remove it if it doesn't fit or seem correct.

**Comment by buddhist-uni** [comment]
For the future: no spaces in the file name please! 😊

**Pull request merged**